# Fluorescence - [work in progress]

Fluorescence serves as a proxy for numerous essential ocean variables ([EOV](https://goosocean.org/what-we-do/framework/essential-ocean-variables/)). Its measurement is to study both biological and physical oceanographic phenomena.

The ["Background"](#background_fluo) section summarizes the general principles of fluorescence, its applications in oceanography, and relevant measurement units and standards. The [Sensing](#sensing_fluo) section covers core optical methods and associated techniques. Fluorescence techniques have led to breakthrough technologies associated with Nobel prizes, which are covered in a brief [historical section](#historical_fluo).

**Contributions**:
- [Olivier Fauvarque](https://annuaire.ifremer.fr/cv/27100/), [Morgane Tardivel](https://annuaire.ifremer.fr/cv/18539/), [Clothilde Haristoy], [Adèle Moncuquet](https://orcid.org/0000-0002-0016-107X)

---
(background_fluo)=
## Background
```{figure} ../_static/fluo/fluo_intro_0.png
:width: 400px
:name: intro_fluo
:align: center

Figure 1 - Different solutions with fluorophores
```
Under specific illumination (usually UV), molecules called fluorophores emit visible light. The color varies depending on the fluorophore (Figure [1](#intro_fluo)). 
Fluorescence is the ability of a molecule to emit light at a different wavelength than the one used to excite it. 


### General principle 

Fluorescence is a nonlinear optical phenomenon where new wavelengths are produced from molecular excitation (Figure [2](#intro_physic)). When a photon with a specific excitation wavelength $\lambda_{Exc}$ is absorbed by a molecule, it promotes the molecule to a higher energy state. From this excited state, non-radiative and radiative transitions spontaneously occur to reduce the molecule's energy level (Figure [3](#physic_Jablonski)). The radiative transition generates light emission at lower energy, corresponding to a longer wavelength $\lambda_{Emit}$. Molecules capable of generating this radiative transition at different wavelengths are called fluorophores. Note that in reality, fluorophores are sensitive to a spectrum of light and emit a spectrum of wavelengths. 

```{figure} ../_static/fluo/fluo_intro.png
:width: 500px
:name: intro_physic
:align: left

Figure 2 - Scheme representing linear (top) and nonlinear (bottom) optics. An Eperlan du large (middle picture) receives light at given wavelengths λ (left picture and colored arrow). The emitted light spectrum (right colored arrow) depends on the incident light. In linear optics, the fish receives visible wavelengths (400-900 nm) and reflects light with the same spectral components and different intensity. Here, the fish has absorbed red, orange, and yellow wavelengths, making it appear purple, blue and green. In nonlinear optics, the fish receives UV light (λ < 400 nm) and emits both reduced UV and green light; this phenomenon is fluorescence.
```

The energy shift, called the Stokes shift, can be important for some fluorescent technique sensitivity [The Molecular Probes® Handbook—A Guide to Fluorescent Probes and Labeling Technologies]. Fluorescence encompasses these three stages, illustrated by Jablonski diagrams (Figure [3](#physic_Jablonski)). The non-radiative transitions introduce a time delay between excitation and emission referred to as the excited lifetime $\tau$, which is typically measured to characterize a fluorophore. 


```{figure} ../_static/fluo/fluo_Jablonski.svg
:width: 250px
:name: physic_Jablonski
:align: right

Figure 3 - Jablonski diagram of absorbance, non-radiative decay, and fluorescence. Electronic transitions are about 1 eV. Absorption is about 1 femtosecond, relaxation takes about 1 picosecond, fluorescence takes about 1 nanosecond. S0 and S1 represent different electronic states. The other numbers (here 0–3 are shown) represent vibrational states. By Jacobkhed from the [Jabonski Wikipedia page](https://fr.wikipedia.org/wiki/Diagramme_de_Jablonski) (french version).
```

All biological and most organic materials fluoresce when excited with UV light [Misra et al. 2025]. The intensity of fluorescence is not constant and depends on the physiological (i.e., physical and chemical) state of the fluorescent material and its environment (pH, Cl⁻ concentration, temperature, CO₂ levels, etc.). In a fluorescent material, only a portion of the excited molecules produce fluorescence emission as other processes may also depopulate the excited state. Quenching refers to any process that decreases fluorescence intensity [Lakowicz 2006]. The ratio between the number of fluorescence photons emitted to the number of photons absorbed measures the fluorescence efficiency and is referred to as quantum yield. Learn more about fluorescence in the [Molecular Probes Handbook](https://www.thermofisher.com/fr/fr/home/references/molecular-probes-the-handbook/introduction-to-fluorescence-techniques.html). In oceanography quenching processes are typicall used to measured envrionmental parameters such as oxygen or pH. Learn more about oxygen and pH measurements in the specific sections.  


```{figure} ../_static/fluo/fluo_materials.jpg
:width: 500px
:name: fluo_materials
:align: center

Figure 4 - Fluorescent minerals under UV light - by Hannes Grobs/AWI, from the french [Fluorescence Wikipedia page](https://fr.wikipedia.org/wiki/Fluorescence).
```

To summarize, the key parameters in fluorescence include:
- The fluorophore 
- Its excitation spectrum
- Its emission spectrum
- The excited state lifetime: $\tau$
- The quantum yield
- Molar absorption coefficient: relates the absorbance to the molecular concentration using Beer-Lambert's law

These parameters depend on the material and its environment. Biological and organic materials have different fluorescence spectra [Fig 2. in Misra et al. 2025] which makes a powerful tool to distinguish species. The intensity also depends on the physiological state of the living organism or on the number of times the sample fluoresces. Fluorescence intensity also depends on environemental parameter such as salinity, oxygen and pH through quenching processes. For instance the more dissolved oygen the less intensity is emitted by a givan sample of fluorophore, this specificity is exploited in optodes.  

### Proxy applications

Fluorescence is a powerful tool that has found widespread applications in human endeavors. Fluorescence measurements are used as proxies in medical diagnostics for cancer detection (https://bmccancer.biomedcentral.com/articles/10.1186/1471-2407-12-21), industrial processes, and research applications (fluorescence research has been recognized with two Nobel prizes: https://www.lemonde.fr/planete/article/2008/10/09/prix-nobel-la-fluorescence-d-une-meduse-primee_1105007_3244.html, https://presse.inserm.fr/cest-dans-lair/prix-nobel-de-chimie-2014/). 

In oceanography, fluorescence serves important functions for marine life such as fish and sharks (https://www.nature.com/articles/srep24751), and biofluorescence is employed to study and differentiate phytoplankton populations and monitor coral health (https://www.nature.com/articles/srep01421). Fluorescent cameras are used to track floating and submerged objects (https://www.nature.com/articles/s41598-025-17302-9) and serve as proxies for pH measurements (https://www.nature.com/articles/s41598-023-37578-z). Fluorescence from colored dissolved organic matter (CDOM) enables tracking of ocean currents and water mass identification: https://www.nature.com/articles/s41598-021-86948-y, https://www.nature.com/articles/srep33978. Different organic components can be distinguished by their unique fluorescence spectra, which proves useful for differentiating Pacific and riverine water masses (adapted from: https://www.nature.com/articles/s41598-021-86948-y). 

### Units and standards

Calibration of fluorescence sensors in oceanographic applications requires rigorous adherence to established standards to ensure data comparability across instruments and research campaigns. For chlorophyll-a fluorescence measurements, primary calibration is typically performed using extracted chlorophyll solutions of known concentration, prepared from cultured phytoplankton or spinach extracts dissolved in acetone (Holm-Hansen et al., 1965; Lorenzen, 1966). Secondary standards include solid-state references or stable fluorescent dyes such as Rhodamine WT, which provides long-term stability for field deployments (Smart et al., 1996; Carlson et al., 2017). CDOM fluorescence is conventionally calibrated against quinine sulfate dihydrate solutions, with results expressed in Quinine Sulfate Units (QSU) or parts per billion (ppb) quinine sulfate equivalents. Instrument-specific calibration coefficients must account for temperature dependencies, non-linear response at high concentrations, and potential quenching effects. Regular verification using certified reference materials and inter-comparison exercises between instruments are essential practices to maintain measurement accuracy and facilitate integration of datasets across different platforms and temporal scales.


### References
- The Molecular Probes® Handbook—A Guide to Fluorescent Probes and Labeling Technologies: accessible on [Thermofisher website](https://www.thermofisher.com/fr/fr/home/references/molecular-probes-the-handbook/introduction-to-fluorescence-techniques.html)


A verifier 
- Holm-Hansen, O., Lorenzen, C. J., Holmes, R. W., & Strickland, J. D. (1965). Fluorometric determination of chlorophyll. ICES Journal of Marine Science, 30(1), 3-15.
- Lorenzen, C. J. (1966). A method for the continuous measurement of in vivo chlorophyll concentration. Deep Sea Research, 13(2), 223-227.
- Smart, A. E., Turley, C. M., & Joint, I. R. (1996). The use of Rhodamine WT as a fluorescent tracer in oceanographic studies. Journal of Plankton Research, 18(9), 1543-1552.
- Carlson, D. F., Özgökmen, T., Novelli, G., Guigand, C., Chang, H., Fox-Kemper, B., ... & Mensa, J. (2017). Surface drifter trajectories from Lagrangian Submesoscale Experiment (LASER). GLAD experiment June–July 2016. Dataset Papers in Science.



---

(sensing_fluo)=
## Sensing

### Optics
Fluorescence signal is very short (less than 1 ms) and can be measured by fast sensor without being covers by daylight.
### Derivation

(historical_fluo)=
## Historical, high-tech sensing and nobel prize

## Available sensors

In [5]:
# https://pandas.pydata.org/pandas-docs/stable/user_guide/style.html 

s = (
    pd.read_csv("sensors.csv", skiprows=0)
    .rename(columns=lambda x: x.strip())
    .set_index("sensor name")
)
# duplicate column to have an artificially wide table
#s = pd.concat([s,s,s], axis=1)
#s = pd.concat([s,]*10, axis=0) 

# can't 
# https://stackoverflow.com/questions/76157621/pandas-sticky-column-headers-using-styler-not-working-sticky-index-rows-works-g
# https://stackoverflow.com/questions/42724327/pandas-dataframe-table-vertical-scrollbars
HTML(
    s
    .to_html(render_links=True, escape=False, na_rep=" ")
    #.style.set_sticky(axis="index").set_sticky(axis="columns")
)

# not satisfactory as it hides rows
#HTML(s.style.set_sticky(axis="columns").to_html(render_links=True, escape=False, max_rows=5))

,technology,measured quantity,absolute accuracy,range,relative accuracy,stability,niveau de validation,lien document de validation,intégration mécanique,intégration électronique interface de communication,Sampling volume,Max. depth,coût,lien doc technique,lien fournisseur,Excitation & Fluorescence wavelengths (nm),Calibration fluorophore
sensor name,,,,,,,,,,,,,,,,,
NaN,,,,,,,,,,,,,170 (2023),,Prosensor,,
Trilux,Optic,Chlorophyll a\nPhycerythrin/Phycocyanin\nNephelometry,,0-100µg/L,,,,,Connecteur Impulse MCIL-6-MS,RS232 / analogique (0-5V),(diam) 26 mm x (long) 60mm,600m,2500 (2023),https://chelsea.co.uk/products/trilux/#product-brochures,https://chelsea.co.uk/,N.C.,
ECO FL,Optic,CHL-A,,,,,,,,RS232,,600m,9000 (2024),https://www.seabird.com/asset-get.download.jsa?id=67383560051,https://www.seabird.com/eco-flntu/,470 / 695 nm,
C3 / C6,Optic,CHL-A,,0-500 µg/L,,,,,,RS232,,600m,,http://docs.turnerdesigns.com/t2/doc/manuals/998-2300.pdf,https://www.turnerdesigns.com/,,
NaN,,,,,,,,,,,,,,,,,
Wimo (NKE),Optic,Chlorophylle A\nPhycocyanine\nPhycoérythrine,0.03 ppb*\n0.1 ppb*\n0.1 ppb*,0-500 ppb*\n0-4500 ppb*\n0-750 ppb*,<1%\n<1%\n<1%,,,,,Mobdus RS232/RS485\nSolution 3G/4G,,250m,,https://nke-instrumentation.com/wp-content/uploads/2022/08/Datasheet-WIMO-FR-web.pdf,https://nke-instrumentation.fr/produit/wimo-sonde-multiparametres/,,*Rhodamine
NaN,,,,,,,,,,,,,,,,,
SmartFluo,quantification de la lumière rouge présente sur l’image prise par le téléphone,Chlorophyll a,1 DN/s ≤ e ≤ 80 DN/s \n(plus d’erreur proche de 1 que de 80 DN/s),Min : 10 µg/L\nMax : 250 µg/L,,"4,5 DN/s",,,,,,Surface,33€ - 73 € + prix du smartphone,https://www.mdpi.com/1424-8220/17/4/678,DIY,,
Fluoromètre Phytoplancton,quantification du voltage transmis après excitation de la chlorophylle avec une led à 448 nm,Chlorophyll a,"e ≥ 0,3 µg/L",Min : 0 µg/L\nMax : 100 µg/L,4 % comparer à un instrument commercial,,,,,UART,,2m,$ 150,https://www.mdpi.com/1424-8220/13/6/7872,DIY,448 nm,


---

**A garder ?**

| Capteur | EOV/EBV | Matériel Principal | Dénomination | Article scientifique | Coût | Dissémination | Profondeur max | Gamme de mesure | Erreur de mesure | Erreur systématique | Capteur de comparaison |
| :-------: | :-----------: | :----------: | :--------------------------: | :--------------------: | :-------------------------------: | :----------: | :-----------: | :-----------: | :-----------: | :-----------: | :-----------: |
| [SmartFluo: A Method and Affordable Adapter to Measure Chlorophyll a Fluorescence with Smartphones](https://www.mdpi.com/1424-8220/17/4/678) | Chlorophylle A | Smartphone | Open-source, Low cost | 2017 | 30€ - 70€ + smartphone | Espagne, Irlande, Allemagne, Norvège | Surface | Min: 10µg/L - Max: 250µg/L | 1DN/s < e < 80DN/s (DN: Digit number) | N.C | LS 55, PerkinElmer (R²= 0.98) |
| [In situ Measurements of Phytoplankton Fluorescence Using Low Cost Electronics](https://www.mdpi.com/1424-8220/13/6/7872) | Chlorophylle A | Arduino | Low cost | 2013 | 150$\$$ | Etats-Unis (rivière) | 2m | Min: 0.3µg/L - Max: 100µg/L | e ≥ 0.3µg/L | 4$\%$ comparer à un instrument commercial | WETStar (R²= 0.886) |
|[A low-cost and portable fluorometer based on an optical pick-up unit for chlorophyll-a detection](https://www.sciencedirect.com/science/article/pii/S0039914023011980)| Chlorophylle A | Arduino, Lecteur DVD | Low cost | 2023 | 140$\$$ | N.C | Surface | Min: 0.35µg/L - Max: 100µg/L | e ≤ 4 µg/L  | < 4$\%$  N.C |
|[A Low-Cost Fluorometer Applied to the Gulf of Saint Lawrence Rhodamine Tracer Experiment](https://ieeexplore.ieee.org/abstract/document/10149195)|Rhodamine| Custom PCB| Low cost | 2023 | 750$\$$ | Quebec | Surface | Min: 0.2µg/L - Max:60µg/L | 0,11mV ≤ e ≤ 0,37mV | 2$\%$ | Turner Cyclops-7 et AML X2Change (R²0.99)|

In [ ]:
Pour la chlorophylle :

Utilisation de LED bleues (450-470 nm) comme source d'excitation
Photodiode avec filtre rouge (>650 nm) pour la détection
Géométrie à 90° ou 180° selon le design
Certains projets utilisent des Arduino ou Raspberry Pi pour l'acquisition

Le "OpenCTD" project a des extensions fluorimètre assez documentées.
PhotosynQ est un projet DIY qui mesure la fluorescence de la chlorophylle pour étudier la photosynthèse.
Des projets utilisent aussi des spectrofluorimètres DIY basés sur des spectromètres miniatures (comme Ocean Optics ou Hamamatsu) combinés avec des sources LED multiples.
Pour le CDOM, des designs avec LED UV (365-385 nm) et détection dans le visible sont courants.